recipes-kernel/linux/files/bcm2712-raspberrypi5-domd.dts

```diff
@@ -34,13 +34,72 @@
 			sdhci-caps = <0x0 0x0>;
 			sdhci-caps-mask = <0xc000 0x0>;
 			xen,path = "/axi/mmc@fff000";
-			xen,force-assign-without-iommu = <0x1>;
+			xen,force-assign-without-iommu;
```

Comment:
This is the correct change, but it should be in the separate commit, as it merely fixes prior mistake.

Reply:
Moved to separate PR